pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -848,6 +848,162 @@ func TestFilterPlugins(t *testing.T) {
+		name       string
+		plugins    []*TestPlugin
+		wantStatus *Status
+	}{
+		{
+			name: "NoBindPlugin",
+			plugins: []*TestPlugin{},
+			wantStatus: NewStatus(Skip, ""),
+		},
+		{
+			name: "SuccessBindPlugins",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Success)},
+				},
+			},
+			wantStatus: NewStatus(Success, "injected status"),
+		},
+		{
+			name: "UnshedulableBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Unschedulable)},
+				},
+			},
+			wantStatus: NewStatus(Error, `bind plugin "TestPlugin" failed to bind pod "/": injected status`),
+		},
+		{
+			name: "SkipBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Skip)},
+				},
+			},
+			wantStatus: NewStatus(Skip, "injected status"),
+		},
+		{
+			name: "ErrorBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Error)},
+				},
+			},
+			wantStatus: NewStatus(Error, `bind plugin "TestPlugin" failed to bind pod "/": injected status`),
+		},
+		{
+			name: "SkipSuccessBindPlugins",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Skip)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj: injectedResult{BindStatus: int(Success)},
+				},
+			},
+			wantStatus: NewStatus(Success, "injected status"),
+		},
+		{
+			name: "SkipUnschedulableAndUnresolvedBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Skip)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj: injectedResult{BindStatus: int(UnschedulableAndUnresolvable)},
+				},
+			},
+			wantStatus: NewStatus(Error, `bind plugin "TestPlugin 1" failed to bind pod "/": injected status`),
+		},
+		{
+			name: "SuccessErrorBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Success)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj: injectedResult{BindStatus: int(Error)},
+				},
+			},
+			wantStatus: NewStatus(Success, "injected status"),
+		},
+		{
+			name: "ErrorSuccessBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Error)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj: injectedResult{BindStatus: int(Success)},
+				},
+			},
+			wantStatus: NewStatus(Error, `bind plugin "TestPlugin" failed to bind pod "/": injected status`),
+		},
+		{
+			name: "UnschedulableAndSuccessBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{BindStatus: int(Unschedulable)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj: injectedResult{BindStatus: int(Success)},
+				},
+			},
+			wantStatus: NewStatus(Error, `bind plugin "TestPlugin" failed to bind pod "/": injected status`),
```

Comment:
Compare to `TestFilterPlugins`, seems don't have cases that cover

```
Success,Success
Error,Error
Success,Unschedulable
```

Should we follow `TestFilterPlugins` and add it as well?

Reply:
There is no harm in more test cases. I will add them